### PR TITLE
Project1-review

### PR DESCRIPTION
**Pull request by jnchaitanya077**

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+@app.route("/userDetails", methods=["POST"])
+def userDetails():
+    firstName = request.form.get("fname")
+    lastName = request.form.get("lname")
+    userName = request.form.get("username")
+    print(firstName, file=sys.stderr)
+    print(lastName, file=sys.stderr)
+    print(userName, file=sys.stderr)
+
+    return render_template("user.html",  user=userName, name=firstName+" "+lastName)
```
Comment (krajeshkumar11) [inline project1/application.py]: These two has to bought into one route to understand how to redirect to same route for both get and post for registration page..

**Comment by krajeshkumar11** [comment]
@jnchaitanya077  good work, work on the provided review and merge with master.

File: project1/application.py (hunk 1 above)
Comment (jnchaitanya077) [reply project1/application.py]: Changed index and redirected to register using redirect.

**Comment by jnchaitanya077** [comment]
Completed.

**Pull request closed without merging**